### ISSUE TITLE
Add spot for (and prompt people to add) Github ids on people pages

### ISSUE THREAD

**Issue by davidlmobley** [openforcefield/openforcefield.org#19]
It dawned on me it can be nontrivial to find people's github IDs to tag them. We should list these on the people pages.

Do we have to do something to the templates to add a spot for it?

**Comment by jchodera** [comment]
1. Add `github_username: xxxxxxx` to the `.yaml` files
2. After [this line](https://github.com/openforcefield/openforcefield.org/blob/master/openforcefield/themes/kube/layouts/section/members.html#L34)
```
        {{ with .google_scholar }} | <a href="{{ . }}">Google Scholar</a> {{ end }}
```
and its friends in the file. add
```
        {{ with .github_username }} | Github: <a href="https://github.com/{{ . }}">{{ . }}</a> {{ end }}
```
3. Re-render!

**Comment by davidlmobley** [comment]
@leeping any chance you have time for this? I'd really like to have everyone's GitHub IDs where I can easily find them as I keep wanting to tag people and not knowing their IDs. :)

**Comment by leeping** [comment]
Should be able to do this. :)

**Comment by jchodera** [comment]
Addressed in #62
